locale/fi/LC_MESSAGES/django.po

```diff
@@ -111,7 +111,7 @@ msgstr "Käyttäjät"
 msgid "Input an URL identifier for this course."
 msgstr "Syötä URL-osoitteessa käytettävä tunnus kurssille."
 
-#: course/models.py:68 course/models.py:729 exercise/exercise_models.py:127
+#: course/models.py:68 course/models.py:729 exercise/exercise_models.py:128
```

Comment:
We will have less git conflicts across different branches and pull requests if these line number comments are not included in the git commit. With `git add -p`, you can add only the relevant lines from one file into the staging area and commit.

Reply:
It's good to include them time to time, but in cases like these where you add a single new translation string, then it's better to include only that part. When there are a lot of other changes too, then it's a good place to update the numbers.

Though, I was supposed to thing this in the style guide, as there is an option not to inlcude line numbers (but the filenames would stay). Should help with this.